easybuild/easyblocks/generic/intelbase.py

```diff
@@ -187,18 +181,7 @@ def configure_step(self):
                 self.license_file = self.cfg['license_file']
             except:
                 # the default should exist
-                self.log.deprecated('No new style license_file parameter, license_file is now mandatory', '2.0')
-                self.license_file = None
-
-            if self.license_file is None:
-                self.log.deprecated('Checking for old style license', '2.0')
-                self.cfg.enable_templating = False
-                lic = self.cfg['license']
-                # old style license is a path (type string)
-                if isinstance(lic, License) and isinstance(lic, str):
-                    self.log.deprecated('No old style license parameter, license has to be pure License subclass', '2.0')
-                    self.license_file = lic
-                self.cfg.enable_templating = True
+                self.log.nosupport('No new style license_file parameter, license_file is now mandatory', '2.0')
```

Comment:
change this to:

```python
self.license_file = self.cfg['license_file']
if not self.license_file:
    self.log.error("No license file specified, yet it must be")
```

don't use it in a `try`, since `license_file` will always be available (if not set in the easyconfig, it'll be the default value)

no reason to still use `nosupport`, just log an error if no license file is specified